export/http.go

```diff
@@ -0,0 +1,94 @@
+package export
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"net/http"
+	"regexp"
+
+	"go.opencensus.io/metric/metricdata"
+	"google.golang.org/protobuf/proto"
+)
+
+// HTTP is an exporter that exports metrics to an
+// HTTP endpoint
+type HTTP struct {
+	address   string
+	apiKey    string
+	apiSecret string
+	headerMap map[string]string
+	client    *http.Client
+	config    Config
+}
+
+// NewHTTP returns a new exporter agent with an HTTP exporter attached
+func NewHTTP(address string, apiKey string, apiSecret string, config Config) *ExporterAgent {
```

Comment:
Should return (*ExporterAgent, error) rather than panic inside the library code.